code/game/objects/items/plushes.dm

```diff
@@ -676,3 +676,23 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+	var/souls = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] begins staring deeply into the eyes of [src]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	if(do_after(user, 50, target = user) && !user.hellbound)
```

Comment:
```suggestion
	if(do_after(user, (5 SECONDS), target = user) && !user.hellbound)
```

Reply:
not sure why there's a need for a do_after for someone who has decided they don't want to play the game anymore. I'd prefer we keep shame ones minimal since the point of the var is to not print le epic unique formatted messages constantly.

Reply:
the plushie can't steal your soul instantly; you need to take the time to look deep into its eyes

plus, it adds to the ritual-like flavor of a group/cult of people who decide to sacrifice themselves to this plushie

Reply:
the point of the verb is not to create le epic cult but to say you don't want to play the round anymore in an "interesting" way.

I'd accept an uninterruptible version (I think it's an arg on one of the do_'s). As long as someone can't delay the user from suiciding after they've confirmed is the goal.

Reply:
Sounds like the intent of having someone take a moment to stare at it before the death will only ever lead to outlier instances, such as someone taking it out of their hands, knocking them over, dragging them around etc. 
Doesn't sound like I'd be able to deal with all of those potential outcomes in a way that "cannot be delayed," so I'll just pull the delay altogether.